cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -504,6 +508,62 @@ export function changePropagateFrames(frames: number): AnyAction {
     };
 }
 
+
+//To remove annotation objects present in given range of frames
+export function removeAnnotationsinRangeAsync(sessionInstance: any, startFrame: number, endFrame: number, force: boolean): ThunkAction{
+    return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
+        const { filters, showAllInterpolationTracks } = receiveAnnotationsParameters();
+        try {
+
+            for(let frame = startFrame; frame<=endFrame; frame++){
+                await dispatch(changeFrameAsync(frame));
+
+                const states = await sessionInstance.annotations.get(frame, showAllInterpolationTracks, filters );
```

Comment:
If ``showAllInterpolationTracks`` is true, it will remove all the tracks in the job, even if they are not presented in range. 
If filters is not empty, it will not remove some objects which are not filtered according to filters.